### PR TITLE
chore: set up new JSX transform and remove "import React"

### PR DESCRIPTION
**Pull request by SanjeevLakhwani**
Set up new JSX transform and remove "import React" from files

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -38,6 +39,7 @@ module.exports = {
       },
     ],
     semi: ['error', 'always'],
+    'react/react-in-jsx-scope': 'off',
```
Comment (v-rocheleau) [inline .eslintrc.js:42]: Looking at the doc for jsx-runtime, Im confused as to why `"react/jsx-uses-react": "off"` is not required here too, any idea?

**Comment by davidlougheed** [comment]
needs build fix

File: .eslintrc.js (hunk 1 above)
Comment (SanjeevLakhwani) [reply .eslintrc.js:42]: no clue

File: src/js/components/Overview/Chart.tsx [hunk 2]
```diff
@@ -1,4 +1,5 @@
-import React, { memo } from 'react';
+// import React, { memo } from 'react';
```
Comment (davidlougheed) [inline src/js/components/Overview/Chart.tsx:1]: rm?

**Review by davidlougheed — CHANGES_REQUESTED** [review]

**Review by davidlougheed — APPROVED** [review]

**Pull request merged**